common/src/main/java/com/wynntils/models/map/pois/SeaskipperPoi.java

```diff
@@ -0,0 +1,220 @@
+        this.endZ = endZ;
+        this.width = endX - startX;
+        this.height = endZ - startZ;
+        this.destinationCenter = new PoiLocation(startX + width / 2, null, startZ + height / 2);
+
+        Vec3 playerLocation = McUtils.player().position();
+
+        isPlayerAtPoi = playerLocation.x > startX
+                && playerLocation.x < endX
+                && playerLocation.z > startZ
+                && playerLocation.z < endZ;
+    }
+
+    @Override
+    public PoiLocation getLocation() {
+        return destinationCenter;
+    }
+
+    @Override
+    public DisplayPriority getDisplayPriority() {
+        return DisplayPriority.HIGHEST;
+    }
+
+    @Override
+    public boolean hasStaticLocation() {
+        return true;
+    }
+
+    @Override
+    public void renderAt(
+            PoseStack poseStack,
+            MultiBufferSource bufferSource,
+            float renderX,
+            float renderY,
+            boolean hovered,
+            float scale,
+            float mapZoom) {
+        poseStack.pushPose();
+        poseStack.translate(0, 0, 100);
+
+        final float renderWidth = width * mapZoom;
+        final float renderHeight = height * mapZoom;
+        final float actualRenderX = renderX - renderWidth / 2f;
+        final float actualRenderZ = renderY - renderHeight / 2f;
+
+        renderedX = actualRenderX;
+        renderedY = actualRenderZ;
+        renderedX2 = renderedX + renderWidth;
+        renderedY2 = renderedY + renderHeight;
+
+        CustomColor color;
+
+        if (isPlayerAtPoi) {
+            color = CommonColors.ORANGE;
+        } else {
+            color = CommonColors.WHITE;
+        }
+
+        BufferedRenderUtils.drawRect(
+                poseStack,
+                bufferSource,
+                color.withAlpha(65),
+                actualRenderX,
+                actualRenderZ,
+                0,
+                renderWidth,
+                renderHeight);
+
+        BufferedRenderUtils.drawRectBorders(
+                poseStack,
+                bufferSource,
+                color,
+                actualRenderX,
+                actualRenderZ,
+                actualRenderX + renderWidth,
+                actualRenderZ + renderHeight,
+                0,
+                1.5f);
+
+        BufferedFontRenderer.getInstance()
+                .renderAlignedTextInBox(
+                        poseStack,
+                        bufferSource,
+                        CodedString.fromString(destination),
+                        actualRenderX,
+                        actualRenderX + renderWidth,
+                        actualRenderZ,
+                        actualRenderZ + renderHeight,
+                        0,
+                        color,
+                        HorizontalAlignment.CENTER,
+                        VerticalAlignment.TOP,
+                        TextShadow.OUTLINE);
+
+        poseStack.popPose();
+    }
+
+    public void renderAtWithoutBorders(
```

Comment:
This method is basically a version of the default render. Perhaps call this in renderAt with the rect border render method.

Reply:
Sorry I'm not understanding what you're suggesting here